vidarr/wdl.py

```diff
@@ -207,6 +239,62 @@ def read_output(output: WDL.Decl):
         'workflow': doc.source_text,
         'accessoryFiles': {
             imported.uri: imported.doc.source_text for imported in doc.imports}}
+    
+    output_meta = doc.workflow.meta.get("output_meta", {})
+
+    modified_output_lines = []
+
+    # Iterate over each output defined in the workflow
+    for output in doc.workflow.outputs:
+
+        # Check if the output name exists in the output metadata
+        if output.name in output_meta:
+            output_metadata = output_meta[output.name]
+
+            if isinstance(output_metadata, dict) and 'vidarr_label' in output_metadata:
```

Comment:
There is no else block for this if, so if an output is in output_meta but it's not a dict (it's the String: String format in fastqc) then it's never added to modified_output_lines, and gets deleted